apps/signing/tests/test_views.py

```diff
@@ -232,9 +232,9 @@ def test_version_exists_with_pk(self):
         newer_upload = FileUpload.objects.latest()
         assert newer_upload != upload
 
-        response = self.get(self.url(self.guid, '3.0', upload.pk))
+        response = self.get(self.url(self.guid, '3.0', upload.uuid))
         assert response.status_code == 200
-        assert response.data['pk'] == upload.pk
```

Comment:
This might cause backwards-compatibility issues. If we care about that we'll need to cheat and pretend that the `uuid` is the `pk` in the response.